include/graph.h

```diff
@@ -1,23 +1,37 @@
-#pragma once
+#ifndef MAIN_GRAPH_H
+#define MAIN_GRAPH_H
 #include <cstdlib>
 #include <exception>
 #include <set>
+#include <fstream>
 #include "supernode.h"
 
+#ifdef VERIFY_SAMPLES_F
+#include "../test/util/deterministic.h"
+#endif
+
 enum UpdateType {
-  INSERT,
-  DELETE,
+  INSERT = 0,
+  DELETE = 1,

```

Comment:
Is this necessary? Aren't these just the default values for an enum. If you have a reason feel free to keep the change.

Reply:
I'd like to keep them for reference since these are the values that show up in the `sample.txt` file for `INSERT` and `DELETE`

Reply:
Okay then I have no problem with this :)